src/main/java/roomescape/member/model/Member.java

```diff
@@ -1,13 +1,29 @@
 package roomescape.member.model;
 
-public class Member {
+import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
 
-    private final Long id;
-    private final MemberRole role;
+@Entity
+public class Member {
 
-    private final MemberEmail email;
-    private final MemberName name;
-    private final MemberPassword password;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(EnumType.STRING)
+    @Column(length = 20, nullable = false)
+    private MemberRole role;
+    @Embedded
+    private MemberEmail email;
+    @Embedded
+    private MemberName name;
+    @Embedded
+    private MemberPassword password;
```

Comment:
JPA 엔티티를 작성하다보면 어노테이션을 붙이는 코드가 많이 생기게 됩니다. 제 기준일지는 모르겠지만 이때 프로퍼티 간에 적절한 개행을 추가해준다면 코드의 가독성을 좀 더 높일 수 있습니다.

```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Enumerated(EnumType.STRING)
    @Column(length = 20, nullable = false)
    private MemberRole role;
    
    @Embedded
    private MemberEmail email;
    
    @Embedded
    private MemberName name;
    
    @Embedded
    private MemberPassword password;
```

Reply:
코드에 적용하였습니다 감사합니다!